quilt/tools/const.py

```diff
@@ -10,6 +10,7 @@ class TargetType(Enum):
     PANDAS = 'pandas'
     FILE = 'file'
 
+UTCF = "%Y-%m-%dT%H:%M:%S"
```

Comment:
The format is not actually specific to UTC, so I'd call it something else...

(Also, apparently, `strftime` supports `%F`, while `strptime` doesn't - despite the docs saying the formats are identical. WTF...)